### ISSUE TITLE
ociobakelut doesn't error if writing to non-existent folder

### ISSUE THREAD

**Issue by dbr** [AcademySoftwareFoundation/OpenColorIO#382]
With OCIO 1.0.9, ociobakelut claims to have written the file even if the output path is not writeable (non-existent parent folder or unsuitable permissions), and incorrectly exits with a zero exit-code

For example:

```
$ ociobakelut --inputspace lin --outputspace project --format houdini --v /tmp/nonexistantdirectory.lut
...
[OpenColorIO INFO]: Wrote '/tmp/nonexistantdirectory.lut'

$ ociobakelut --inputspace lin --outputspace project --format houdini --v /root/test.lut
...
[OpenColorIO INFO]: Wrote '/root/test.lut'
```

**Comment by YashAgrawal0** [comment]
Is this issue still open?

**Comment by doug-walker** [comment]
Yes, I believe this is still a useful issue to work on Yash.

**Comment by YashAgrawal0** [comment]
@dbr  @doug-walker  

When I run this command : 

>  ociobakelut --inputspace lin --outputspace project --format houdini --v /tmp/nonexistantdirectory.lut

I got the following errors :  

```
[OpenColorIO Info]: Color management disabled. (Specify the $OCIO environment variable to enable.)
OCIO Error: Error baking houdini:Could not find source color space 'lin'.
```

Can you help me resolve these?

**Comment by dbr** [comment]
Oh, “lin” and “project” are color spaces in our internal OCIO config, but they can be changed to any other space and the problem be the same.

You’d need to download one of the example configs (or write one - the actual content of the colour spaces don’t have any bearing on the error so long as they are syntactically valid), then point the command to that config file either via the OCIO end variable or the `--iconfig` argument). The docs on opencolorio.org should cover this (can’t link to them as Github pages is currently down for some reason)

**Comment by YashAgrawal0** [comment]
In my machine I ran ociobakelut command for both the cases when output file path is specified in a non-existent folder and in a directory with root permissions, in both the cases I got the error  `non-writable path` which was the expected behaviour.

Refer to the [image](https://user-images.githubusercontent.com/24262584/78782486-be801180-79bf-11ea-9a7b-53a733231ca4.png) for more info.

**Comment by dbr** [comment]
Ahh! I only just noticed this is already fixed in 01246674361fcee31be8aba2eee74722f1f018c7 via pull request #854
